packages/headless/src/user.ts

```diff
@@ -60,14 +64,23 @@ export function useUpdateUser({
   const update = useCallback(
     async ({
       id,
-      payload,
+      userDataPayload,
+      topicsPayload,
     }: {
       id: number;
-      payload: IUser.UpdateApiPayload;
+      userDataPayload: IUser.UpdateApiPayload;
+      topicsPayload?: ITopic.ReplaceUserTopicsApiPayload;
     }) => {
-      return await atlas.user.update(id, payload);
+      // TODO: Handle Updating profile Pictures here
```

Comment:
Let's add it in this PR or create task for it on clickup.

Reply:
added a task for it in click up?
is there an endpoint for uploading photos?

Reply:
Yes, you can upload it using `atlas.user. updateMedia`.